pkg/browsers/console.go

```diff
@@ -26,14 +26,8 @@ var ServiceMap = map[string]string{
 	"":               "console",
```

Comment:
GitHub won't let me suggest a change as the imports aren't changed in this file, but you'll need to remove the `"sort"` import as it's no longer used and causes the build to break (Go is very strict on unused imports)